src/renderer.ts

```diff
@@ -264,15 +264,29 @@ export const render = async (
   const normalized = normalizeJsx(node);
   const pres = new pptxgen();
 
-  let layout = "LAYOUT_16x9";
-  if (normalized.layout === "16x10") {
-    layout = "LAYOUT_16x10";
-  } else if (normalized.layout === "4x3") {
-    layout = "LAYOUT_4x3";
-  } else if (normalized.layout === "wide") {
-    layout = "LAYOUT_WIDE";
+  // https://gitbrent.github.io/PptxGenJS/docs/usage-pres-options/#custom-slide-layouts
+  if (Object.keys(normalized.customLayout || {}).length > 0) {
+    const { width = 0, height = 0 } = normalized.customLayout as {
+      width: number;
+      height: number;
+    };
+    pres.defineLayout({
+      name: "customLayout",
```

Comment:
I wonder if it would be better if the width and height were coded into the name here, so that multiple custom layouts would function together. Something like
```
name: `customLayout-${width}x${height}`
```